benchmarks/01_trivial/05_packed_load_store.ispc

```diff
@@ -0,0 +1,63 @@
+export uniform int width() { return programCount; }
+
+// This case has mask set all_on.
+#define PACKEDLOAD_ALL(T_ISPC)                                                                                         \
+    export uniform int32 packed_load_##T_ISPC##_all(uniform T_ISPC *uniform src, uniform T_ISPC *uniform dst,          \
+                                                    uniform int count) {                                               \
+        uniform int32 num = 0;                                                                                         \
+        foreach (i = 0...count) {                                                                                      \
+            varying T_ISPC var = 0;                                                                                    \
+            num += packed_load_active(&src[num], &var);                                                                \
+            dst[i] = var;                                                                                              \
+        }                                                                                                              \
+        return programCount;                                                                                           \
```

Comment:
Did you mean to return `num`?

Reply:
Ahh...I tried a slightly diff signature. It's not needed since I added the 'Width()' function. Should be 'void' return now

Reply:
I meant to keep it as a void return. Anyway I modified it to return num and check it in the cpp side.

Reply:
It needs to be consumed. Either returned or stored somewhere. Otherwise popcnt logic will be removed by dead code. Returning value, on the other side, will require adopting test for GPU. But we'll need to have a dedicated effort for that anyway.